### ISSUE TITLE
Plugin should support latest Eiffel event version

### ISSUE THREAD

**Issue by e-pettersson-ericsson** [jenkinsci/eiffel-broadcaster-plugin#6]
I noticed that the events being sent by this broadcaster plugin is from an older version of Eiffel protocol, the Toulouse version. I think it would be wise to update to support the Agen edition of Eiffel protocol.

I propose to either:
* step the version in the existing events to align with Agen version
* add another version of Eiffel events being supported (configurable)

I'm not sure how a suden change would affect current users of the plugin, could it be possible to support several protocol versions? And let the users choose in the plugin configuration?

**Comment by Isacholm** [comment]
We should step the protocol-version to the latest.

In case of letting the version be configurable,  I think it's problematic. The reason being that if for some reason the protocol has fundamental changes like we had in the 2.0 version where GAV was changed to purl for identifying Artifacts, you had to support both cases which are alot different from each other just to support a legacy version of the protocol. Maybe this complexity is worth it, what do you think?

If it is worth it, I guess you could add legacy support at a later time.

**Comment by e-pettersson-ericsson** [comment]
I agree it is easiest to only step to latest version of protocol and not support several versions. In my view it is not worth the complexity but I guess it depends on current users. They might have to prepare before they upgrade the plugin to newer version, but as long as it is clearly communicated I don't see the problem.

**Comment by magnusbaeck** [comment]
The removal of the ArtC/ArtP events (#8) will be a backwards incompatible change that warrants a bump to 2.x so we can slip in a protocol upgrade while we're at it.
